### DISCUSSION TITLE
Sourcer les impacts

### DISCUSSION THREAD

**Discussion by yannkozon** [cnumr/best-practices#103] [General]
La graduation des impacts est parfois difficile à appréhender voir même potentiellement fausse.
Exemple : "Utiliser les notations CSS abrégées", "Préférer les CSS aux images" avec un impact fort.

**Comment by ACTLEM** [comment]
C'est un point qui va être retravaillé avec le passage de 3 à 5 niveaux. Tu as raison ;-)
